src/main/java/sirius/web/http/Response.java

```diff
@@ -678,6 +702,27 @@ public Response infinitelyCached() {
         return this;
     }
 
+    /**
+     * Sets the value for the {@link #CUSTOM_PROXY_CACHE_TTL_HEADER} header.
+     *
+     * @param ttl the value to set
+     * @return <tt>this</tt> to fluently create the response
+     */
+    public Response withCustomProxyTTL(String ttl) {
+        this.customProxyTTL = ttl;
+        return this;
+    }
+
+    /**
+     * Sets the default value for the {@link #CUSTOM_PROXY_CACHE_TTL_HEADER} header.
+     *
+     * @return <tt>this</tt> to fluently create the response
+     */
+    public Response withDefaultCustomProxyTTL() {
+        this.customProxyTTL = defaultCustomProxyTTL;
+        return this;
+    }
```

Comment:
**Optional:** Correct Camel-Casing would be `withCustomProxyTtl`

Reply:
It is, but looks *ugly* 🤓